src/main/java/duke/Event.java

```diff
@@ -0,0 +1,79 @@
+package duke;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Event class is a task. The input must be in such a format
+ * "Event <event name> /at <event date>".
+ */
+public class Event extends Task{
+    protected String description;
+    protected boolean isDone;
+    final String EVENT = "[E]";
+    protected String dateAndTime;
+    protected LocalDateTime localDateTime;
+    DateTimeFormatter dtf = DateTimeFormatter.ofPattern("MMM dd yyyy HH:mm");
+
+    /**
+     * Public constructor for an event.
+     *
+     * @param description Description of the event.
+     * @param dateAndTime Date and time of the event.
+     */
+    public Event(String description, String dateAndTime) {
+        super(description);
+        this.description = description;
```

Comment:
I am not sure if this double description assignment is necessary since there is a description variable for Task already and the super(description) already assigns the description

Reply:
I agree with @Ongzl , you can still get the subclass instance's `description` (from parent class) without the subclass having a private field for `description`.

Reply:
Thanks for pointing this out, will do the changes!